recipe/build.sh

```diff
@@ -27,4 +27,10 @@ rm -rf torchvision/csrc/io/image/cpu/giflib
 export TORCHVISION_USE_FFMPEG=0
 export TORCHVISION_USE_NVJPEG=${FORCE_CUDA}
 export TORCHVISION_INCLUDE="${PREFIX}/include/"
+
+# disabled by default but with a TODO to "enable by default"
+export TORCHVISION_USE_AVIF=1
+# (typo upstream)
+export TORCHVISION_USE_HEIC=1
```

Comment:
Not exactly a good sign for the maturity of upstream support. 😅

Reply:
Actually, I was being presumptions. Apparently "HEIC" is a [specific codec](https://en.wikipedia.org/wiki/High_Efficiency_Image_File_Format#HEIC:_HEVC_in_HEIF), so the files are called `.heif` or `.heic`. My mistake.